src/lib/Theme.ts

```diff
@@ -318,6 +319,8 @@ export default class Theme {
                 }
             });
             Logger.success('Theme syncing DONE...');
+            let url = `${AVAILABLE_ENVS[currentContext.env]}`
+           let domain = url.replace("api.","");
```

Comment:
Use URL parsing instead of using string replace for getting a domain name. @anuragpandey1115